Source/WebCore/css/calc/CSSCalcValue.cpp

```diff
@@ -339,7 +339,10 @@ bool CSSCalcValue::equals(const CSSCalcValue& other) const
 
 inline double CSSCalcValue::clampToPermittedRange(double value) const
 {
-    if (primitiveType() == CSSUnitType::CSS_DEG && (isnan(value) || isinf(value)))
+    value = CSSCalcOperationNode::convertToTopLevelValue(value);
+    // If an <angle> must be converted due to exceeding the implementation-defined range of supported values,
+    // it must be clamped to the nearest supported multiple of 360deg.
```

Comment:
Why is 0 the nearest supported multiple of 360deg to infinity and -infinity?

Reply:
I guess it would be 360 or -360 which are equivalent to 0

Reply:
That’s not literally correct (360deg would not be the nearest multiple of 360deg to infinity), but if this is modulo 360deg then clearly any multiple is equivalent to 0deg. But that was not obvious to me.